ext/civicrm_admin_ui/ang/afsearchGrantStatus.aff.html

```diff
@@ -0,0 +1,17 @@
+<div class="af-container">
```

Comment:
Likewise shouldn't this move into the civigrant extension or is this extension going to cover grant stuff as well @colemanw ?

Reply:
this too has been moved to the extension, thanks for the review.